acceptance-tests/src/test/java/tech/pegasys/ethsigner/tests/AcceptanceTestBase.java

```diff
@@ -34,8 +34,8 @@
 
 public class AcceptanceTestBase {
```

Comment:
It's going to make more sense to have a separate `TestBase` for Hashicorp, as that'll be a better pattern moving forward then a single base that caters for every integration.

Reply:
Will do that once that there is more than one test class using Hashicorp Vault